CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
+* AKS will no longer support the [GPU image (preview)](https://github.com/Azure/AKS/issues/4472) to provision GPU-enabled AKS nodes. Starting on Jan 10, 2025 you will no longer be able to create new GPU-enabled node pools with the GPU image. Alternative options that are supported today and recommended by AKS include the default experience with manual NVIDIA device plugin installation or the NVIDIA GPU Operator, detailed in [AKS GPU node pool documentation](https://learn.microsoft.com/azure/aks/gpu-cluster?tabs=add-ubuntu-gpu-node-pool#confirm-that-gpus-are-schedulable).
+* Starting on January 1, 2025, [invalid values sent to the Azure AKS API for the properties.mode field of AKS AgentPools will be rejected](https://github.com/Azure/AKS/issues/4468). Prior to this change, unknown modes were assumed to be User. The only valid values for this field are the (case-sensitive) strings:["User", "System"](https://learn.microsoft.com/rest/api/aks/agent-pools/create-or-update?view=rest-aks-2024-02-01&tabs=HTTP#agentpoolmode), or ["Gateway"](https://learn.microsoft.com/rest/api/aks/agent-pools/create-or-update?view=rest-aks-2024-06-02-preview&tabs=HTTP#agentpoolmode).
+
+### Release Notes
+
+* Features:
+  * [Trusted launch](https://learn.microsoft.com/azure/aks/use-trusted-launch), which improves the security of generation 2 virtual machines (VMs) by protecting against advanced and persistent attack techniques is now generally available.
+  * AKS patch versions `1.30.7`, `1.29.11`, `1.28.13`, `1.27.21`, `1.26.22`, and `1.25.24` are now available. Refer to [version support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy) and [upgrading a cluster](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli) for more information..
+
+* Preview features:
+  * AKS version [`1.31`](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/) is now available in preview. 
+  * You can now specify the default Seccomp profile applied to all workloads by setting the `seccompDefault` on [customized node configuration](https://learn.microsoft.com/azure/aks/custom-node-configuration?tabs=linux-node-pools#linux-kubelet-custom-configuration).  value of `RuntimeDefault` will apply the default from the container's runtime which restricts certain system calls for improved security.
+  * You can now [specify the GPU driver type (preview)](https://learn.microsoft.com/azure/aks/use-windows-gpu) when creating a new AKS Windows GPU Nodepool using the `--driver-type` flag.
+    
+* Bug fixes:
```

Comment:
Can we add Github links to all the bug fixes please ?

Reply:
These are not on GiHub.